examples/nlp/question_answering/question_answering_squad.py

```diff
@@ -26,12 +28,22 @@
 def main(cfg: DictConfig) -> None:
     logging.info(f'Config: {cfg.pretty()}')
     trainer = pl.Trainer(**cfg.trainer)
-    exp_manager(trainer, cfg.get("exp_manager", None))
+    log_dir = exp_manager(trainer, cfg.get("exp_manager", None))
+    infer_datasets = [cfg.model.validation_ds, cfg.model.test_ds]
```

Comment:
infer_datasets or eval_datasets?

Reply:
The goal for this code section is to set the correct path for the prediction file. That is why infer_datasets

Reply:
Ah makes sense. But right now it's not being added properly as we don't assign back the updated dataset list ?

Reply:
Or I guess just the path needs to be updated inside the config without assigning the value itself.

Reply:
infer_datasets is a mutable type (dict), in the following line i change the dict values